tomni/cytosmart_data_format/annotations/polygon/main.py

```diff
@@ -168,5 +206,38 @@ def _calculate_roundness(self) -> None:
             self._calculate_area()
 
         _, radius = cv2.minEnclosingCircle(self._contour)
-        enclosing_circle_area = radius ** 2 * np.pi
+        enclosing_circle_area = radius**2 * np.pi
         self._roundness = self._area / enclosing_circle_area
+
+    @staticmethod
+    def __compare_list_points(l1: List[Annotation], l2: List[Annotation]):
```

Comment:
Seems more like `def compare_list_points(l1: List[Point], l2: List[Point]):` kind of thing